compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -67,11 +67,25 @@ internal class ComposeSceneMediator(
     composeSceneFactory: (ComposeSceneMediator) -> ComposeScene,
 ) {
     private var isDisposed = false
+    private val invisibleComponent = InvisibleComponent()
 
-    val skiaLayerComponent by lazy { skiaLayerComponentFactory(this) } // TODO: Make private
-    val contentComponent get() = skiaLayerComponent.contentComponent
+    val skikoView: SkikoView = DesktopSkikoView()
 
-    private val invisibleComponent = InvisibleComponent()
+    private val platformComponent = DesktopPlatformComponent()
+    private val textInputService = DesktopTextInputService(platformComponent)
+    private val _platformContext = DesktopPlatformContext()
+    val platformContext: PlatformContext get() = _platformContext
+
+    private val skiaLayerComponent by lazy { skiaLayerComponentFactory(this) }
+    val contentComponent by skiaLayerComponent::contentComponent
+    var transparency: Boolean
```

Comment:
```suggestion
    var transparency: Boolean = useInteropBlending
        set(value) {
            field = value
            skiaLayerComponent.transparency = value
        }
```

and in `init`:
```
skiaLayerComponent.transparency = this.transparency 
```

Otherwise, this isn't consistent:
```
mediator.transparency = false
println(mediator.transparency) //prints true
```

Reply:
> and in init:
> 
> `skiaLayerComponent.transparency = this.transparency`

It's already there.

Actually, getter isn't used anywhere, so I'll convert this property to function later.